README.md

```diff
@@ -0,0 +1,32 @@
+## <u>Progressive Streaming</u>
+
+A simple application that allows the tranfer of a video file from the server to the client and upon download, has playback. Bonus addition of a transcoder script as well. Ffmpeg libav API is used for the decoding and transcoding the video file.
+
+### To run the script
+Make sure you have a directory named bin in your project home to run the following commands.<br>
```

Comment:
This could be automated, through the makefile no need to make the dev go through a simple hoop.

Reply:
yeah once the makefile is done I will remove those, is that fine?

Reply:
yeah